.pre-commit-config.yaml

```
@@ -13,19 +13,26 @@ repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
     rev: v0.3.5 # must match requirements-tests.txt
     hooks:
-      - id: ruff
-        name: Run ruff on stubs, tests and scripts
-        args: ["--exit-non-zero-on-fix"]
       - id: ruff
         # Run this separately because we don't really want
         # to use --unsafe-fixes for all rules
+        # Should be run first as it can leave unused imports behind
         name: Remove unnecessary `sys.version_info` blocks
         args: ["--exit-non-zero-on-fix", "--select=UP036", "--unsafe-fixes"]
+      - id: ruff
+        name: Run ruff on stubs, tests and scripts
+        args: ["--exit-non-zero-on-fix"]
       - id: ruff
         # Very few rules are useful to run on our test cases;
         # we explicitly enumerate them here:
         name: Run ruff on the test cases
-        args: ["--exit-non-zero-on-fix", "--select=FA,I,RUF100", "--no-force-exclude", "--unsafe-fixes"]
+        args:
+          [
+            "--exit-non-zero-on-fix",
+            "--select=FA,I,RUF100",
+            "--no-force-exclude",
+            "--unsafe-fixes",
+          ]
```

not a "YAML expert", but isn't this the more normal way of doing a multiline list in YAML? 🤨

```suggestion
        args:
          - "--exit-non-zero-on-fix"
          - "--select=FA,I,RUF100"
          - "--no-force-exclude"
          - "--unsafe-fixes"
```

---

I have no strong feelings either way (including reverting this line). It's just a line above 90 chars autoformatted. The way you're suggesting is more concise but wouldn't get shrunk back by a formatter if params are removed.

---

I don't mind too much either way whether we leave it as it is or reformat it, but the way you have it currently looks kinda weird to me ;) so I'd rather either revert it or apply my suggestion. IMO it's not a massive issue if a formatter would fail to shrink the line back if params were removed in the future. Perfect formatting of our YAML files isn't exactly a _priority_ :)

---

Then I'll apply your suggestion, preferring to avoid long lines.